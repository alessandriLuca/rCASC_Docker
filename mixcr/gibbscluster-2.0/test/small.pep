TLDSDQVGY HLA-A01:01
FTDTEGLPY HLA-A01:01
YLLPQQPPPSL HLA-A02:01
NLIANVLYL HLA-A02:01
YILRELPKV HLA-A02:01
SLLDIDPLI HLA-A02:01
ATDIFSYLVE HLA-A01:01
SSDEEFAAALY HLA-A01:01
GTVYEDLRY HLA-A01:01
ATDVLHAVWY HLA-A01:01
YSQSPAYNSQY HLA-A01:01
LTETIYKNYY HLA-A01:01
VLQGKLAEV HLA-A02:01
LLWPGAALLV HLA-A02:01
YLEEYKFQV HLA-A02:01
GLLGYLHFV HLA-A02:01
TLIRVPIPQV HLA-A02:01
FTEVLSPNMY HLA-A01:01
ISYGPDWKDFY HLA-A01:01
FSSDDSKLEY HLA-A01:01
NSDVSLLNY HLA-A01:01
ALMPVLNQV HLA-A02:01
VLDRLTFDY HLA-A01:01
LLDNSYYEQY HLA-A01:01
YLWNTVSKL HLA-A02:01
VLIKDLFHI HLA-A02:01
NLLKLIAEV HLA-A02:01
AMFGKLMTI HLA-A02:01
LLESAKMFNY HLA-A01:01
ALYASRLYL HLA-A02:01
YLKELLNLA HLA-A02:01
TIMKKVTSV HLA-A02:01
FIEALLPHV HLA-A02:01
TVDVLKEAY HLA-A01:01
YLKEILEQL HLA-A02:01
KLQLVEMPL HLA-A02:01
ALLETMFGV HLA-A02:01
LADFGFGNF HLA-A01:01
AVYEGHVSCV HLA-A02:01
FLDPRPLTVV HLA-A02:01
LSDTKYLEMI HLA-A01:01
ILLDRLLYL HLA-A02:01
ALYSAEHGL HLA-A02:01
FTEDSLLRH HLA-A01:01
LLLAAPAQA HLA-A02:01
LTDLVLKSSVY HLA-A01:01
LMDKSSTIY HLA-A01:01
LLLDMQNRL HLA-A02:01
TLQEFLERI HLA-A02:01
VADVLVDGY HLA-A01:01
KLLEPVPVSV HLA-A02:01
LMNPNIASV HLA-A02:01
DVYADHLDWHY HLA-A01:01
TLIDSLDTL HLA-A02:01
MLDWASGTLY HLA-A01:01
HLLNESPML HLA-A02:01
HLEEGKDLEY HLA-A01:01
VVDNLQHLY HLA-A01:01
RMLEDRDLFVM HLA-A02:01
NSELGSETRY HLA-A01:01
KLFSGVLMDL HLA-A02:01
SLLESVQKL HLA-A02:01
KTDPVRLTQLY HLA-A01:01
YIDAQFENY HLA-A01:01
TIDLGAAAHY HLA-A01:01
TSDFIPRLVY HLA-A01:01
HTDHHMFFDY HLA-A01:01
YLAKLIFQM HLA-A02:01
LSEYSKFLSY HLA-A01:01
YLAERIPTL HLA-A02:01
TSEPKPFLYY HLA-A01:01
FQDQLHQLY HLA-A01:01
KLLDGPPGV HLA-A02:01
HIAEVSQEV HLA-A02:01
TLEESAMLRY HLA-A01:01
VLERNPNGWWY HLA-A01:01
PTDLETRSYKY HLA-A01:01
YVDEYRNDY HLA-A01:01
FLLEKGYEV HLA-A02:01
LLYEDIPDKV HLA-A02:01
FMDKLGENL HLA-A02:01
LLLKYTEKL HLA-A02:01
ASEPSAQLSY HLA-A01:01
ITSTRSLNY HLA-A01:01
YTDAPPAYSEL HLA-A01:01
YTDNLVRVW HLA-A01:01
LLLPDYYLVT HLA-A02:01
ATDIFSYLV HLA-A01:01
FLTRIISQL HLA-A02:01
GLSSLSIHL HLA-A02:01
FLNDIFERI HLA-A02:01
FLNQVLAEI HLA-A02:01
TADGKTYYY HLA-A01:01
ILNEIVNFV HLA-A02:01
FLNGGILNYMI HLA-A02:01
TTEFLYNLY HLA-A01:01
YLLGLGPLEV HLA-A02:01
ALNNLLHSL HLA-A02:01
LQDPFHGVY HLA-A01:01
YLFDEIDQAL HLA-A02:01
ALYTGDPNL HLA-A02:01
SLYGGTITI HLA-A02:01
YSDSAVLKL HLA-A01:01
YLAEKYEWDVA HLA-A02:01
YADIAVNSDRY HLA-A01:01
YLDGRLEPLY HLA-A01:01
ETVPRSGEVY HLA-A01:01
FSDFLGLHSK HLA-A01:01
LLDRFLATV HLA-A02:01
TTDFEDDEF HLA-A01:01
YVDDTQFVGFD HLA-A01:01
NTDLWIVMEY HLA-A01:01
SSEVTSQLY HLA-A01:01
TSDHFETIMK HLA-A01:01
QTAEASHVSRY HLA-A01:01
TLIELWEGLTV HLA-A02:01
ATDRFRSLY HLA-A01:01
QTENSLHMGY HLA-A01:01
SLLPEGPPAI HLA-A02:01
SLAESLDQA HLA-A02:01
ATNPESKVFY HLA-A01:01
LTEADLQYGY HLA-A01:01
TSEQNPPYSY HLA-A01:01
DTSRPLGDSMY HLA-A01:01
FLLPKFHGV HLA-A02:01
YVDEELPDY HLA-A01:01
ETDPVENTY HLA-A01:01
LMEDEMKEYY HLA-A01:01
RADDADEFGY HLA-A01:01
LTDIKLGKY HLA-A01:01
SLDDFQAWL HLA-A02:01
FVDDYTVRV HLA-A01:01
TFNLEGSLIY HLA-A01:01
FLWERPTLL HLA-A02:01
MSEERTLLY HLA-A01:01
ALQEKLQYL HLA-A02:01
ALARTALAL HLA-A02:01
FLLFINHRL HLA-A02:01
NTDPYKRAVY HLA-A01:01
YTDFDGTRVYS HLA-A01:01
ALADGVVSQA HLA-A02:01
KLLCGLLAERL HLA-A02:01
SSEYHYVGGFY HLA-A01:01
SLDGKAIKV HLA-A02:01
MLDEILLQL HLA-A02:01
VIDEDVKSGRY HLA-A01:01
VLFGLLREV HLA-A02:01
ATAAALTYGLY HLA-A01:01
PTEISQSLLY HLA-A01:01
VLEINSLKY HLA-A01:01
RLGEAEGLMKV HLA-A02:01
KLQNLNIFL HLA-A02:01
HLYSEVKEV HLA-A02:01
LVDNSIRELQY HLA-A01:01
KLFIGGLNV HLA-A02:01
KLGDLFFRV HLA-A02:01
ASDPNSSIF HLA-A01:01
LTEVNKTGTQY HLA-A01:01
SLFGGSVKLA HLA-A02:01
DLDLESPKSLY HLA-A01:01
FLGDGMGV HLA-A02:01
VADDPNDPY HLA-A01:01
TTEAVHYLY HLA-A01:01
KLNPQQFEV HLA-A02:01
GILGIQPPSV HLA-A02:01
MVESLVGWGY HLA-A01:01
QSERQLLLY HLA-A01:01
LIDVGTGYY HLA-A01:01
LSDIGSGIRYL HLA-A01:01
FTDKAASY HLA-A01:01
FIEERLLMY HLA-A01:01
ILNPLLTLL HLA-A02:01
YLEEDVYQL HLA-A02:01
LVEKSQLGY HLA-A01:01
ITDYLIEEV HLA-A01:01
TLGSFSGLRV HLA-A02:01
VLGEGVPIL HLA-A02:01
FMYDRPLRL HLA-A02:01
GLDNINVFL HLA-A02:01
LLLPDYYLV HLA-A02:01
KVASLLHQV HLA-A02:01
ALLEQTGDMSL HLA-A02:01
LMELTMLDY HLA-A01:01
ALPEIFTEL HLA-A02:01
TLEGTPVFV HLA-A02:01
TMADQIVTV HLA-A02:01
AMNGHTAAV HLA-A02:01
FLLWATAEA HLA-A02:01
VVEDKVEGY HLA-A01:01
ETEVSFLDY HLA-A01:01
YINASFLDGY HLA-A01:01
ELFSSPPAV HLA-A02:01
FVDERPEEV HLA-A02:01
KQVSDLISV HLA-A02:01
NTDFYQTSY HLA-A01:01
KTDMEKLTFY HLA-A01:01
FTASWGDLHY HLA-A01:01
KLQIVEMPL HLA-A02:01
VLTEHVAAA HLA-A02:01
FLSLINVGLI HLA-A02:01
