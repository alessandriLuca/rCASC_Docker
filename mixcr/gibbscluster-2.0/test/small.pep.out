#Call: /Users/massimo/Applications/gibbscluster-2.0_DB/gibbscluster-2.0/GibbsCluster-2.0e_SA.pl -f small.pep -P gibbs -I 1 -D 5 -g1-4 -S3
#Mon Apr  3 09:37:01 2017
#Session ID: 4516
#Run name: gibbs_4516
#Read 200 unique sequences
##### Settings: #####
#Shift moves and cluster moves are activated
#Number of clusters: 1 - 4
#Motif length: 9
#Initial MC temperture: 0.8
#Number of temperature steps: 20
#Number of iterations x Sequence x Tstep: 100
#Max insertion length: 1
#Max deletion length: 5
#Interval between Indel moves: 10
#Interval between Single Peptide moves: 20
#Interval between Phase Shift moves: 100
#Number of initial seeds: 3
#Penalty lambda: 0.8
#Weight on small clusters: 5
#Preference for hydrophobic P1: 0
#Sequence weighting type: 0
#Use trash cluster to remove outliers: 0
#### Running Gibbs clustering...
#Clustering with 1 groups, seed number 1
#Clustering with 1 groups, seed number 2
#Clustering with 1 groups, seed number 3
#Clustering with 2 groups, seed number 1
#Clustering with 2 groups, seed number 2
#Clustering with 2 groups, seed number 3
#Clustering with 3 groups, seed number 1
#Clustering with 3 groups, seed number 2
#Clustering with 3 groups, seed number 3
#Clustering with 4 groups, seed number 1
#Clustering with 4 groups, seed number 2
#Clustering with 4 groups, seed number 3
#Clustering complete!
### Determining seeds with highest KLD...
#Best 1 groups, seed 1
#Best 2 groups, seed 2
#Best 3 groups, seed 1
#Best 4 groups, seed 3
###Parsing result files...
#RESULTS for 1 CLUSTERS
#1 Final Average KLD: 9.653264
#1	1	200	9.653
#RESULTS for 2 CLUSTERS
#2 Final Average KLD: 11.669350
#2	1	99	13.450
#2	2	101	9.924
#RESULTS for 3 CLUSTERS
#3 Final Average KLD: 11.469785
#3	1	100	13.248
#3	2	6	8.072
#3	3	94	9.795
#RESULTS for 4 CLUSTERS
#4 Final Average KLD: 10.694555
#4	1	100	13.109
#4	2	3	3.146
#4	3	19	6.817
#4	4	78	8.834
